exporter/opentelemetry-exporter-jaeger-proto-grpc/setup.cfg

```diff
@@ -44,7 +44,7 @@ install_requires =
     grpcio >= 1.0.0, < 2.0.0
     googleapis-common-protos ~= 1.52
     opentelemetry-api ~= 1.3
-    opentelemetry-sdk ~= 1.3
+    opentelemetry-sdk ~= 1.11
```

Comment:
Should this be 1.11.1?

Reply:
~1.11 will pick the latest 1.* version and gives the flexibility later which would not be possible with hard == 1.11.1